kubeflow/trainer/rhai/transformers_instrumentation.py

```
@@ -0,0 +1,1450 @@
+                    # Verify storage access by writing/reading test file (if enabled)
+                    if checkpoint_config.get("verify_cloud_storage_access", True):
+                        test_file = ".kubeflow-access-test"
+                        last_error = None
+                        for attempt in range(1, 4):  # 3 attempts with backoff
+                            try:
+                                self.remote_fs.pipe(test_file, b"test")
+                                self.remote_fs.cat(test_file)
+                                self.remote_fs.rm_file(test_file)
+                                last_error = None
+                                break  # Success
+                            except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)